### DISCUSSION TITLE
Dicas para melhorar o Inglês?

### DISCUSSION THREAD

**Discussion by kazzkiq** [frontendbr/forum#109] [Discussão]
Recentemente eu tenho pesquisado muito sobre vagas no exterior e/ou trabalho remoto, e uma dúvida que sempre bate é: **Como eu vou me sair no Inglês?**

O problema não é escrever ou conseguir me comunicar em si, até porque eu acho que ~~quase~~ todo mundo que trabalha com computação tem essa capacidade. O receio é sobre não conseguir ter uma desenvoltura tão boa (ou pelo menos próxima) à nossa língua nativa, ou de não conseguir me expressar tão bem e acabar virando motivo de chacota e piada entre os nativos (como já foi relatado em outra Issue), etc.

Tendo isso em vista, eu queria saber o que a galera do grupo faz para aprimorar o Inglês. Cursos pagos? Algum chat aberto? Vídeo-aulas? Jogos online? E aí, quais as dicas pra melhorar a pronúncia e não parecer um robô falando inglês? :smile: :us:

**Comment by milacarrera** [comment]
Assisto o máximo de coisas possíveis em inglês sem legenda (filmes, series, documentários, qualquer coisa) e ouço muito podcast e rádios de fora para treinar o listening.
Para speaking, tem alguns sites que você pode encontrar pessoas nativas/fluentes para conversar. O que eu mais uso são gospeaky.com e interpals.net.
Ainda sim eu travava um pouco para falar porque morria de medo e vergonha, mas o que ajudou mesmo foi fazer intercâmbio!

**Comment by rickbenetti** [comment]
Eu também estou nesta pegada @kazzkiq estou buscando vagas na minha terra [Bolzano](http://www.bolzano-bozen.it/), mas lá Italiano e Alemão devem ser básicos, ter o inglês faz toda a diferença, inclusive em outras parte da Europa, e estou com dificuldade de encontrar grupos para chat em inglês.

**Comment by rickbenetti** [comment]
@milacarrera uma coisa que faço é assistir filme com legendas, mas no idioma do filme, ou seja Inglês - Inglês, Espanhol - Espanhol, Francês - Francês e por aí vai.

**Comment by milacarrera** [comment]
@rkb81 Isso, no inicio eu usava legendas também, mas depois de um tempo é bom tirar, mesmo que no começo nós fiquemos perdidos. Afinal quando formos falar com alguém na vida real, não vai ter legenda para ajudar rsrs.

Se o listening não estiver bom ainda, eu recomendo demais esse podcast que é voltado para quem está aprendendo inglês: https://www.eslpod.com/website/

**Comment by leticia** [comment]
Algo que pode ajudar - se te sentires disposto - é ser um host de couchsurfers/Airbnb. Tu acabas convivendo com outras culturas e pessoas e vai te acostumando a pensar em outra língua. 

Se estiveres em uma empresa, podes tentar institucionalizar um English friday, pra que a conversa técnica flua também. 

Update: e tem o famoso [Toastmasters](https://www.toastmasters.org/), também, que tu podes ver se existe na tua cidade.

**Comment by kazzkiq** [comment]
@rkb81 Minha familia por parte de mãe toda fala Alemão, só eu vacilão que não aproveitei pra aprender enquanto era novo. Já daria pra trampar na Alemanha, ou Suíça (Suíça também fala alemão, né?).

@milacarrera Bons sites! Vou me cadastrar e dar uma testada. Sobre os filmes, só arrisco ver sem legenda se eu já assisti antes. Uma vez tentei assistir um lançamento sem legendas, e o sotaque dos atores era tão carregado que parecia que eles falavam uma terceira língua haha. Não entendia nada. Mas realmente não tem nada mais gratificante que ouvir uma música ou ver um vídeo em que você consegue entender a pessoa falando de forma nativa e "normal" (sem falar de vagar, etc).

@leticia Na última empresa em que trabalhei o gerente chegou a implementar a ideia de english day, mas a maior parte da diretoria _cagava_ pra regrinha e falava em português com todo mundo, aí acabava que ninguém levava a sério. Triste.

Mas os dias que funcionou, foi muito legal! Recomendo muito pra quem tiver oportunidade de fazer isso na sua empresa. Mas já aviso que a produtividade cai um pouco, uma conversa que levaria uns 30 segundos virava 3 minutos de trava-língua. 😆

**Comment by vitorabner** [comment]
Essa dica que a @leticia deu é muito boa. Fiz isso através de um programa da faculdade e cara, meu inglês decolou. Eu falo muita coisa errada, porém perdi a vergonha e já consigo conversar numa boa. 

Uma dica legal para intercâmbio é o  https://www.worldpackers.com/. Você trabalha em troca de hospedagem, comida e etc. Em breve irei testar.

**Comment by milacarrera** [comment]
Essa dica do @vitorabner é muito boa também!! Uma colega da faculdade, assim que se formou, se mandou para a Argentina e ficou trabalhando por 2 meses em um hostel por esse World Packers para melhorar o espanhol. Parece que funcionou! haha :)

**Comment by felipefialho** [comment]
Cara, comecei a fazer muito Duolingo ano passado, é ótimo para aprender algumas regras e palavras. Também instalei um app chamado "Aprenda Inglês" que achei ótimo pra listen e speak. 

Apesar de ler diversos artigos e escrever em inglês, não é o suficiente. Minha comunicação é muito ruim, como a maioria das pessoas que não são fluentes: speak e listen.

Sendo assim chutei o balde e fiz algo que achei que nunca faria, me matriculei em um curso de inglês. Faz 3 meses e to achando bem legal até agora.

Claro que não faz milagres, é necessário treinar todos os dias, já que são só duas aulas por semana. Mas até o fim do ano vamos ver se rompo algumas barreiras.

**Comment by willycamargo** [comment]
Eu busco ler todo dia e escrever sempre que posso. Escuto alguns podcasts (3 ou 4 vezes o mesmo até entender hahaha), assisto séries em inglês com a legenda em inglês.

O que eu sofro meeesmo é o "speaking". Eu sei qual é a palavra, eu escuto o meu "cérebro" pronunciando a palavra da maneira correta mas quando eu abro a boca ela sai errado, é foda hahaha

**Comment by kazzkiq** [comment]
> @willycamargo: Eu sei qual é a palavra, eu escuto o meu "cérebro" pronunciando a palavra da maneira correta mas quando eu abro a boca ela sai errado, é foda hahaha

Resumiu de forma super precisa o meu problema. Hahaha

@LFeh Eu queria começar um OpenEnglish ou EnglishTown da vida pra ver se é isso tudo mesmo que dizem. Mas os valores parecem ser bem salgadinhos, e eu desconheço alguém que tenha feito pra dar opinião se vale a pena e tal. Aí fico com receio de perder grana com isso. Você ta fazendo algum desses serviços online, ou escolas presenciais?

**Comment by robsonbittencourt** [comment]
Eu faço Duolingo há um bom tempo, mas chega uma hora que parece que você não avança mais. 

Quanto a cursos e afins, recomendo darem uma olhada nos videos do [Mairo Vergara](https://www.youtube.com/playlist?list=PLl_7J18zHKKA4Kw0wAVVl0wpr31_uqSes). Não são aulas, são dicas sobre aprendizado. Ele também tem um curso de inglês, mas se não estou enganado, abre turma somente umas duas vezes por ano.

**Comment by baltazarparra** [comment]
Filmes e series com audio e legenda em ingles. Ate se sentir confortável e remover as legendas. Codeschool é massa também, pegue algum assunto que ja domine e veja o curso em ingles sem legenda pra reforçar o ingles instrumental.

**Comment by Jessica7** [comment]
Olá pessoal! estou utilizando esse app:
  **HelloTalk**: http://www.hellotalk.com/#pt : focado na conversação, gramática...
Gostei muito, tem vários recursos para praticar o idioma.
**Lang-8**:  http://lang-8.com/ : focado na escrita
É só  escrever um texto no idioma que tá aprendendo, e depois esse  texto é revisado e corrigido por algum nativo do idioma. Adorei tive 90% dos meus textos corrigidos! gostei muitíssimo.

.

**Comment by felipefialho** [comment]
@kazzkiq

Preferi presencial, por vários motivos. 

Primeiro porque trabalho remoto e fico muitas horas no computador, então é uma oportunidade de desconectar.

Também para facilitar as interações, você meio que é forçado a interagir com os outros alunos (em inglês).

Comecei a fazer no Centro Britânico, do lado de casa. A mensalidade está em torno de 340 reais. Pesquisei bastante antes e parece ser uma boa escola.

**Comment by lagden** [comment]
alguém já fez open english ou english town... cursos online?
se sim, o que está achando?

**Comment by bukinoshita** [comment]
Minhas dicas seriam:

### Listening:
- Filmes
- Seriados
- Vlog
- YouTube em geral
- Podcast

### Speaking
- Ler artigos **gringo**, **em voz alta** e **gravar**. (você vai ver o quão sotaque você ainda tem e algumas palavras que você não sabe a pronúncia certa. Caso use mac, ele tem o "Speech" quando clica com botão direito na palavra / google tradutor pra pronuncia, caso não tenha mac).
- Fingir ser gringo, comprar um café ou pedir informação. Além de fazer você se sentir mais confortável ao falar em outra língua, irá também praticar listening e speaking.

### Writing
- Ler artigos :)
- Escrever um blog
- Escrever comentários em tudo, blog, youtube, facebook. Por mais que seja "bobeira", forçará a escrever em inglês e treinar mais a teoria.

### LBNL
- Duolingo: Nunca me ajudou, mas pra quem ta começando pode ajudar e acho os produtos e experiência deles bem legais.
- Viagem
- Curso

**Comment by renatoagds** [comment]
Acho que ter contato com nativos é a melhor solução, principalmente nativos que estão abertos a ajudarem, tem muita galera que gosta da troca de conhecimento, você ensina português para ela e ela te ensina inglês, também, existem muitos professores particulares nativos aqui no Brasil, essa é uma outra alternativa muito boa.

**Comment by nvrossett** [comment]
Cara tem uma série de vídeos bem legais no Youtube
English For You, é focado em conversação, todos só falam inglês o tempo todo mas vc entende tudo.

Canal: https://www.youtube.com/user/englishforyoutv

São 80 Aulas de cerca de 1 hora cada. 
Ou seja, 80h de Aula

Iniciante (30 videos): 
https://www.youtube.com/playlist?list=PLAie3YzOYRmley8mMYy1s9XuZYVxW1Ot7
Intermediário (20 videos): 
https://www.youtube.com/playlist?list=PLAie3YzOYRmkIuEDUwyNmx1THFI7ouQHZ
Elementar (30 videos): 
https://www.youtube.com/playlist?list=PLAie3YzOYRmlNFK7bE7_WppF7QCcOrQ0M

Assiste os primeiro 20 minutos pra você ter uma noção, é tudo por associação, muito legal.

Depois da uma espiada como está o diálogo das últimas aulas, ainda to tentando ter tempo de ver tudo, parei na metade do iniciante e tava curtindo muito até então.

Tem esses canais do youtube também que são bem legais.
https://www.youtube.com/channel/UCgZu_Xuh7YnQn9uAHrHLcDg
https://www.youtube.com/channel/UCcNm9fM9V5wf-0PZVmkM08g
https://www.youtube.com/channel/UC7L6zRg6wm7CT21Rfj44yQA
https://www.youtube.com/channel/UCVBErcpqaokOf4fI5j73K_w

**Comment by hevertoncastro** [comment]
Me indicaram esse aplicativo esses dias:

![image](https://cloud.githubusercontent.com/assets/7852959/14401256/48e34f18-fde1-11e5-94d8-313d4a5016ca.jpeg)

Nele você escolhe um tópico e faz uma ligação gratuita (inicialmente limitado a 5 minutos) para uma pessoa de algum lugar do mundo. 

Como nele tem diversos assuntos e não apenas aprendizado de inglês, dá pra encontrar até nativos. 

![image](https://cloud.githubusercontent.com/assets/7852959/14401267/d48b57b8-fde1-11e5-9c51-de4f464978dc.jpeg)

E a regra é falar inglês, do contrário pode tomar ban. É bem legal. 

No meu caso a vergonha de falar, mesmo com um total desconhecido, ainda atrapalha. 

Tem um muito bom pra treinar listening, é o English Radio:

Ele é muito completo, além de rádios do mundo todo, tem um agrupado de conteúdo, inclusive áudios gravados por nativos e os textos dos áudios. Lá também tem chat de áudio, mas dá muito Chinês com sotaque carregado. Recomendo para listening a rádio NPR, eles falam um pouco mais cadenciado, ótimo para treinar quando se é iniciante. 

![image](https://cloud.githubusercontent.com/assets/7852959/14401282/cc49591e-fde2-11e5-9718-14f62e6a4290.jpeg)

Desculpem as imagens grandes, estou pelo celular e não deu pra reduzir.

**Comment by tcelestino** [comment]
Comecei alguns meses atrás curso com professora particular e vem ajudando muito, você fica mais solta. Por ser uma turma pequena, fica mais "fácil" falar e aprender com os erros. Tento ao máximo participar de discussões pela internet, para melhora a escritar e uma realmente ver vídeos (séries, palestras etc...) também ajuda bastante.

**Comment by luisfmsouza** [comment]
Todas as dicas são validas.

Mas o que realmente vai fazer a diferença @kazzkiq é você pegar suas coisinhas e **se mandar pra um país que fala inglês** e estudar lá. Já que além das horas dentro da sala de aula, você vai precisar se virar pra pedir informação, ir em restaurante, supermercado, bares e etc. Eu fiz isso, e recomendo.

Isso serve pra melhorar seu inglês, óbvio. E pra derrubar os mitos da vergonha e dificuldade de falar/escutar.

**Comment by bertelledani** [comment]
Outra dica legal além do que o pessoal já comentou sobre música, filmes e séries é ler livros em inglês do estilo que mais te agrade, preferencialmente no kindle. O legal do kindle é o dicionário, que você seleciona qualquer palavra e ele te mostra o significado. Isso ajuda a aprender muitas palavras novas e contexto no qual são utilizadas. Você pode começar com livros infanto juvenis que tem vocabulário mais simples e ir progredindo conforme se sentir mais confortável.

![amazon_kindle_paperwhite_upgrade](https://cloud.githubusercontent.com/assets/4275940/15377173/06d3ad70-1d30-11e6-98fa-5499052ba445.jpg)

Se o seu inglês já é intermediário e você puder fazer um intercâmbio, recomendo fortemente! A experiência de estar imerso no uso da língua faz uma diferença gigantesca e você evolui muito rápido. Depois do terceiro mês você já terá muita desenvoltura para conversar sem travar. Porém quando estiver morando fora fuja dos que falam português, para não ter escapatória e ter que usar inglês até nas coisas mais banais, aproveitando melhor o seu tempo.

Ah, sobre filmes e séries, tente consumir conteúdo que tenha sotaques diferentes do que estamos acostumados a ver no Brasil ("hollywoodianos") e procure algo que seja britânico, australiano, irlandês... Isso vai melhorar muito seu listening e também vocabulário. Essa dica é especial para você que quer morar e trabalhar fora, pois muito provavelmente vai precisar lidar com pessoas do mundo inteiro falando inglês cada um com seu sotaque do país de origem!

**Comment by grippado** [comment]
Já pensaram em marcar tipo um micro-meetup com essa galera daqui, que ta estudando e tentando aprender, pra treinar o 'speaking'?
Algo informal mesmo, tipo, umas 4 ou 5 pessoas, marcam em uma starbucks, ou num bar, ou algo do tipo, e do início ao fim desse 'meeting', somente falar ingles!
Acho que seria uma forma de começar a perder a vergonha e forçar pra que o speak melhore!
Eu toparia algo assim, "easily"!

**Comment by napalmdev** [comment]
Caramba, ótimas dicas ai hein, no meu caso, eu comecei a utilizar "tempos mortos" que eu tinha no meu dia pra treinar inglês, por exemplo:

De manhã enquanto estou passando um café :) pra ir trabalhar, eu costumo deixar o podcast do https://www.eslpod.com/ rolando, então no meio do dia, eu costumo fazer ao menos uma revisão de expressions, phrasal verbs e afins com o sistema de repetição espaçada no Anki.
Quando vou e volto do trampo e quando vou e volto da facu eu costumo assistir Friends com com audio e legenda em Inglês, e quando chego na facu eu combinei com um camarada pra gente ir todo dia na biblioteca pra treinar Inglês, então conversamos ao menos 30min por dia,  o que está me ajudando d+.

Mas acredito que eu precise falar com nativos tbm pra melhorar, então vou dar uma olhada nesses links que vcs postaram ai  :)

**Comment by Marcelosilva10** [comment]
Um site excelente para quem quer aprender inglês com seu seriado predileto, https://ororo.tv/en.

**Comment by felipefialho** [comment]
@grippado Nunca pensei nisso, mas acho legal a ideia. Não quer ficar a frente disso?

**Comment by napalmdev** [comment]
@grippado é uma boa ideia mesmo, e se vingar podemos organizar uns hangouts até, pra quem não puder comparecer pessoalmente. É bom começar esse treino de speaking principalmente pra quebrar o gelo, pq muita coisa vc sabe, entende, mas na hora de falar, vc trava, e pra conseguir passar por isso, só conversando mesmo  :D

**Comment by danilovaz** [comment]
@hliberato o que vc tava comentando hoje sobre inglês

**Comment by vitor-mendrone-deel** [comment]
@grippado topo fácil, vamos organizar isso aí? 

Lembrei de um post do @igorapa sobre a experiência dele indo aprender inglês nos EUA. O post é bem detalhado, inclusive sobre os custos. 

http://www.igorapa.com/fui-aprender-ingles.html

http://www.igorapa.com/minha-viagem-aos-eua.html

Vale a pena dar uma lida.

**Comment by Marcelosilva10** [comment]
Um dos sites que ajudou muito foi o duolingo, a escola que o @LFeh e muito renomada as aulas deles são de excelente qualidade, tem uma metodologia que poderia fazer e marcar reuniões ou encontro de amigos e assim todos irem aprofundando o vocabulário, outro site parecido com o Duolingo e o https://www.busuu.com/, estou fazendo aula particular, sou iniciante na área como alguns já sabe mais mesmo sendo iniciante eles pedem inglês intermediário e bom praticar dia a dia.

**Comment by santosluiz** [comment]
Complementando a resposta do @BuKinoshita, você pode utilizar chats online e conhecer pessoas de qualquer lugar do mundo com o mesmo objetivo que você, aprender um novo idioma.

Além de você praticar a escrita, com desenvoltura pode convidar as pessoas para conversar via vídeo pelo Skype. Assim, você pratica a escrita e a fala. Te recomendo estes dois sites:

www.speaky.com
www.howdoyou.do

**Comment by Werlenson** [comment]
Umas dicas que tenho encontrado por aí é assistir seriados, documentários e até mesmo reportagens em inglês. Ultimamente tenho seguido algumas dicas do [Jerry Costa](http://renercampos.com/ingles-do-jerry-costa-aprender-ingles-online/) e tenho achado de muito valia o que ele ensina em alguns vídeos disponíveis em seu canal no YouTube. Mesmo sabendo falar bem pouco, ainda tenho medo de pronunciar o que falo. Qualquer dica para resolver essa situação é de grande ajuda.

**Comment by jonasporto** [comment]
Eu to nessa caminhada também, estou focado em conseguir um upgrade, porque só conseguia ler e escrever(às vezes com auxilio do google tradutor =/), E sempre que precisava falar, não conseguia me comunicar, porque ficava muito nervoso e travava demais.

O que eu tenho feito:

* Duolingo, não parei até finalizar todas as lições, depois abandonei =)
* Ouvir podcast gringos: https://allearsenglish.com/, https://devchat.tv/, http://bikeshed.fm/ demorou um pouco pra começar entender, mas insistindo repetidas vezes, consegui notar uma boa evolução com listening.
* Assistir series em inglês s/ legenda.
* Ler textos em inglês em voz alta.

Pro-Tips:
* Treine [shadowing](https://www.youtube.com/watch?v=GVWFGIyNswI) quando estiver assistindo series ou ouvindo podcasts, é cansativo mas achei a técnica interessante.
* Se puder investir, o https://italki.com/ é excelente pra conversar com diferentes professores ao redor do mundo, alguns disponibilizam uma aula experimental(30 minutos) com um preço bacana. Também é possível conversar gratuitamente com as pessoas da comunidade, mas como nunca consigo me planejar e agendar nada, eu acabo pagando para um professor que esteja disponível na hora que eu quero praticar.

**Comment by michaelfdias** [comment]
Há tempos eu venho estudando inglês e nessa jornada eu fiz cursos (presencial e online), usei aplicativos e conheci ferramentas. Vou passar um pouco do que encontrei de bacana, espero que seja útil.

Eu gosto de utilizar o canal [engVid](https://www.youtube.com/user/engvidenglish), do YouTube, para assistir aulas de inglês, em inglês. Seria como aulas de inglês em um curso de intercâmbio.

Para treinar a compreensão de inglês com músicas ou trechos de filmes eu curto o [LyricsTraining](http://lyricstraining.com).

Quando eu conheço alguma palavra nova e quero ver exemplos de uso dela: [Youglish](http://youglish.com).

Sobre cursos online o melhor que eu encontrei até o momento foi o [ABAEnglish](http://www.abaenglish.com).

Se você busca apenas conversar para treinar a comunicação, eu indico o [Cambly](https://www.cambly.com).
Aqui você conversa online com algum professor nativo, sobre o assunto que quiser, o dia e hora que quiser.
O preço é cobrado baseado no tempo de conversa e a quantidade de vezes por semana. Com R$ 196,00, por exemplo, você já pode treinar 15 minutos todos os dias.
O legal é que a plataforma disponibiliza um histórico de gravação de todas conversas, caso queira rever sua performance rs.

**Comment by leovivanco** [comment]
Boas dicas galera, anotei algumas.

Gostaria de compartilhar também.

[Engoo](https://engoo.com.br/) é um uber do professor de inglês, você escolhe seu plano e tem direito de fazer aulas todos os dias. Recomendo muito, faço desde novembro e raramente deixo de fazer uma aula.

**Anki**, como já foi falado aqui é repetição espaçada, faço com as palavras mais usadas do inglês e verbos irregulares no passado. Mas com ele você consegue aprender qualquer coisa rs.

**Filmes** assistir filmes que você nunca viu e que tenha muito interesse em assistir, mas sem legendas, quando começa gerar um interesse ou uma emoção seu cérebro começa trabalhar em outra frequência e o aprendizado é mais rápido.

**Comment by ingridrauany** [comment]
@leovivanco o rendimento das aulas sendo apenas de 25min é bom? Estou com vontade de assinar o plano, mas não posso pegar muitas aulas devido ao preço ainda.

Acho o **Duolingo** muito bom para aprender palavras, finalizei todas as etapas e fico revisando para relembrar muitas coisas. E tem uma sessão de tradução de textos muito boa também.

**Comment by leovivanco** [comment]
@ingridrauany  a longo prazo vc sente a diferença, principalmente quando vc faz todos os dias inclusive nos fds. Sobre professores as vezes você acaba escolhendo algum ruim, mas com o tempo você vai favoritando os que vc mais gostou. Pelo custo beneficio recomendo muito.

**Comment by suissa** [comment]
Teremos 1 curso de conversação na Webschool, @lucastafarelbs conte mais!

**Comment by expalmer** [comment]
Aproveitando a issue, quais sites em inglês vocês recomendam ? Tipo site de assuntos gerais, notícias, geek ...

**Comment by vicainelli** [comment]
http://www.macmillandictionary.com/

**Comment by vicainelli** [comment]
@expalmer esses são os que tenho no meu feed

- https://www.smashingmagazine.com/
- http://hackingui.com/
- http://www.lukew.com/
- http://www.bbc.com/future/
- https://www.fastcompany.com/
- https://singularityhub.com/
- https://github.com/blog
- https://www.wired.com/

**Comment by eubond** [comment]
Pessoal, sei que ele já foi citado, mas não vi esse conteúdo.

http://www.mairovergara.com/livro/

Agora esse ebook dele é free, no livro ele te explica **como as pessoas aprendem novos idiomas** e como ele criou o método dele em cima disso. A grande sacada então, é que você mesmo pode adaptar o seu estudo (caso não se adapte ao que ele recomenda) com base nas informações de **como aprender** e não só do método do cara de como estudar.

Eu achei bem relevante e interessante. Recomendo.

**Comment by ribeiroevandro** [comment]
Uma coisa que tenho tentado fazer, nos últimos tempos não com frequência é usar o Duolingo, me ajudou bastante... Tentei ver filmes e séries em inglês no Netflix, mas são poucos filmes com legendas em inglês no catalogo brasileiro, acho isso bem ruim... Mas já abri alguns links aqui e vou ver as opções que deixaram e continuar os estudos...

**Comment by dionarodrigues** [comment]
Galera, há muitas dicas sensacionais aqui!

Usei algumas técnicas já compartilhadas aqui - e também outros métodos - para fazer o meu inglês dar um salto em alguns meses. Deixei um comentário sobre como melhorei o meu conhecimento da língua nessa outra issue: https://github.com/frontendbr/forum/issues/478#issuecomment-607836454

Espero ter ajudado! :)

**Comment by BrazilianEagle** [comment]
Olá amigo companheiro de aprendizado.  

Aprender ou aperfeiçoar seu inglês não precisa ser um desafio dispendioso. Aqui estão algumas dicas valiosas, que fui adquirindo e me ajudaram muito ao longo do tempo:
	1.	Cinema em Inglês, a Sessão que Você Nunca Esquecerá:
	◦	Mergulhe em filmes e séries em inglês. Não apenas você se entretém, mas absorve a autenticidade do idioma e aprende expressões cotidianas de forma natural.
	
	2.	Viaje Pelas Páginas de Livros em Inglês:
	◦	Deixe-se envolver por histórias cativantes em inglês. A leitura amplia seu vocabulário, melhora a gramática e proporciona uma compreensão mais profunda da cultura anglo-saxã.
	
	3.	Podcasts - O Seu Companheiro de Aprendizado:
	◦	Ouça podcasts em inglês durante suas atividades diárias. É uma maneira eficiente de treinar seu ouvido para diferentes sotaques e contextos linguísticos.
	
	4.	A Imensidão do YouTube para o Seu Progresso:
	◦	Explore canais educativos em inglês no YouTube. Há uma riqueza de conteúdo para todos os níveis de habilidade, desde gramática até dicas para pronúncia.
	
Estas são apenas algumas dicas para atingir a maestria do inglês, mas certamente uma orientação personalizada e dedicada encurtará bastante esse caminho! 

Nesse caso, recomendo o professor que fiz aula por um bom tempo Giovanni. Ele oferece um ambiente de aprendizado acolhedor e eficaz, possibilitando que você adquira fluência em 8 meses de aula.

Entrando em contato, pode falar que foi recomendado pelo "Brazilian Eagle"! Ele vai saber quem é. haha   


Abraço.
